pkgs/development/python-modules/imageio/default.nix

```diff
@@ -3,25 +3,28 @@
 , fetchPypi
 , pillow
 , psutil
+, imageio-ffmpeg
 , pytest
 , numpy
 , isPy3k
+, ffmpeg
 , futures
 , enum34
 }:
 
 buildPythonPackage rec {
   pname = "imageio";
-  version = "2.4.1";
+  version = "2.5.0";
 
   src = fetchPypi {
-    sha256 = "0jjiwf6wjipmykh33prjh448qv8mpgngfi77ndc7mym5r1xhgf0n";
+    sha256 = "1bdcrr5190jvk0msw2lswj4pbdhrcggjpj8m6q2a2mrxzjnmmrj2";
     inherit pname version;
   };
 
-  checkInputs = [ pytest psutil ];
+  checkInputs = [ pytest psutil imageio-ffmpeg ffmpeg ];
   propagatedBuildInputs = [ numpy pillow ] ++ stdenv.lib.optionals (!isPy3k) [
     futures
+    imageio-ffmpeg
```

Comment:
Why is this only added for Python 2? And you should decide on whether to propagate it even though it's an optional dependency or only use it in `checkInputs`.

Reply:
This is a bit difficult: `imageio-ffmpeg` only works on Python 3. `imageio` works (without ffmpeg) with 2 and 3. But I cannot disable the ffmpeg tests easily (nor do I really want to). I could disable _all_ tests for Python 2, but that's bad, too.

Reply:
But you're adding it only when building for Python 2. You can conditionally add it to `checkInputs` but I'd completely leave it out of `propagatedBuildInputs` in that case.